File: Simple-OTT/app/src/main/java/com/theoplayer/demo/simpleott/datamodel/SimpleOTTConfiguration.java
package com.theoplayer.demo.simpleott.datamodel;

public class SimpleOTTConfiguration {

    public Config config;

    public static class Config {
        public Live live;
        public OnDemand onDemand;
        public Offline offline;
    }

    public static class Live {
        public AssetItem[] channels;
    }

    public static class OnDemand {
        public  AssetItem[] vods;
    }

    public static class Offline {
        public AssetItem[] vods;
    }



}

File: simple-ott/app/src/main/java/com/theoplayer/demo/simpleott/datamodel/SimpleOTTConfiguration.kt
package com.theoplayer.demo.simpleott.datamodel

class SimpleOTTConfiguration {
    var config: Config? = null

    class Config {
        var live: Live? = null
        var onDemand: OnDemand? = null
        var offline: Offline? = null
    }

    class Live {
        lateinit var channels: Array<AssetItem>
    }

    class OnDemand {
        lateinit var vods: Array<AssetItem>
    }

    class Offline {
        lateinit var vods: Array<AssetItem>
    }
}